release-please-config.json

```diff
@@ -0,0 +1,22 @@
+{
+    "bootstrap-sha": "1354831baeafb52965a8b2f5f80df5c43e414c99",
+    "exclude-paths": [
+      ".git",
+      ".idea",
+      ".github",
+      ".vscode"
+    ],
+    "release-type": "dotnet",
+    "include-component-in-tag": false,
+    "include-v-in-tag": true,
+    "packages": {
+      ".": {
+        "package-name": "src/Microsoft.Graph/",
+        "changelog-path": "CHANGELOG.md",
+        "extra-files": [
```

Comment:
We're missing the version file

Reply:
@shemogumbe I can see that you've resolved the comment but the version file is still missing

Reply:
Is the Csproj and the changelog the only places we need to update versions, https://github.com/microsoftgraph/msgraph-sdk-dotnet/pull/2562/commits/1a84bbba79c66f638757e2844f2670f468983980

Reply:
right, because dotnet is actually able to get the information from the assembly metadata, contrary to any other language out there.
https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/31eb169bee9537ecbaacd5a33b93e507ef5bc5ad/src/Microsoft.Graph/GraphServiceClient.cs#L27

Resolving